pandas/tseries/period.py

```diff
@@ -355,6 +355,36 @@ def freqstr(self):
         return self.freq
 
     def asfreq(self, freq=None, how='E'):
+        """
+        Convert the PeriodIndex to the specified frequency `freq`.
+
+        Parameters
+        ==========
```

Comment:
Can you use `---` instead of `===` for the lines? (to be consistent with the other docstrings)